wsi/Android.mk

```diff
@@ -108,6 +108,7 @@ LOCAL_CPPFLAGS += \
 endif
 
 LOCAL_CPPFLAGS += -DENABLE_ANDROID_WA
+LOCAL_CPPFLAGS += -DPREFER_ABGR
```

Comment:
Add Board config check here like i mentioned, you can use ifeq ($(strip $(BOARD_CURSOR_WA)), true)

Reply:
If this this require anther PR in Android stack

Reply:
It doesn't, BOARD_CURSOR_WA is already set their. That's why using this instead of adding a new one.